server/src/main/java/org/uiautomation/ios/utils/SimulatorSettings.java

```diff
@@ -56,7 +60,7 @@
   private InstrumentsVersion instrumentsVersion;
 
   public SimulatorSettings(HostInfo info, String sdkVersion, boolean is64bit, DeviceType device,
-                           DeviceVariation variation) {
+                           DeviceVariation variation) throws NotActiveException {
```

Comment:
I'd prefer something other than `NotActiveException` and something closer to the use case and condition. I was looking for an existing exception that is in ios-driver (and one that was unchecked vs, checked) and `SessionNotinitializedException` stood out. I'm open to other exception suggestions here though.